src/pages/OrderBook.tsx

```diff
@@ -132,16 +143,11 @@ const OrderBook: React.FC = () => {
           <InputBox>
             <label>Hops</label>
             <Input
-              value={hops}
+              value={hopsValue}
               type="number"
               min="0"
               max={ORDER_BOOK_HOPS_MAX.toString()}
-              onChange={(e: ChangeEvent<HTMLInputElement>): void => {
-                const hopsValue = e.target.value
-                if (hopsValue && !Number.isNaN(Number(hopsValue))) {
-                  setHops(Number(hopsValue))
-                }
-              }}
+              onChange={(e: ChangeEvent<HTMLInputElement>): void => setHopsValue(e.target.value)}
```

Comment:
could/should create a handler that accepts `e` as default

```const handleChange = (e: EventType): void => setHopsValue(e.target.value)```

then just pass it as 
```onChange={handleChange}```

Reply:
I usually like to do as you suggest, however in this case, the code is so small, I decide against separating the handler from it's basic implementation. I believe it's still easy to read (it does one thing)

Let me know if u feel strong about this one